apps/files/src/components/Collaborators/NewCollaborator.vue

```diff
@@ -165,15 +155,11 @@ export default {
           })
           const remotes = recipients.exact.remotes.concat(recipients.remotes)
 
-          const results = users.concat(groups, remotes).filter(collaborator => {
-            const selected = this.selectedCollaborators.find(
-              selectedCollaborator => {
-                return (
-                  collaborator.value.shareWith ===
-                  selectedCollaborator.value.shareWith
-                )
-              }
-            )
+          this.autocompleteResults = users.concat(groups, remotes).filter(collaborator => {
+            const selected = this.selectedCollaborators.find(selectedCollaborator => {
+              return collaborator.value.shareWith === selectedCollaborator.value.shareWith &&
+                parseInt(collaborator.value.shareType, 10) === parseInt(selectedCollaborator.value.shareType, 10)
```

Comment:
I thought we already parsed `shareType` to int in our last developments. But ok to keep it for safety

Reply:
Those are coming directly from the SDK, not from the store. So those are not parsed. There is only one occurrence a few lines down, where I was able to use an already parsed `shareType` (changed it now after you pointed it out, thanks!).